kura/org.eclipse.kura.net.admin/src/main/java/org/eclipse/kura/net/admin/visitor/linux/IfcfgConfigReader.java

```diff
@@ -474,6 +491,22 @@ private static void setNetConfigIP4(NetConfigIP4 netConfig, boolean autoConnect,
         }
     }
 
+    private static List<? extends IPAddress> getDhcpRouters(String interfaceName, IPAddress address) {
+        List<IPAddress> routers = null;
+        if (address != null) {
+            DhcpClientLeases dhcpClientLeases = DhcpClientLeases.getInstance();
```

Comment:
This approach makes it more difficult to test. Consider e.g. `protected DhcpClientLeases getDhcpClientLeases()` or make it a static or injectable variable.